tb_ctl.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
Please add a few words into one of readme's when this scripts can/should be used.

Reply:
Line 11 contains message that user can see when use -h option.

Reply:
Please rename to `./run_client_server.sh`. I think it describes more precisely what scripts actually does.
Please add as an optional step into `client_server_mode.md`.
Please provide some sanity results into PR desciption. E.g.,
```
./exec.sh -i client pytest --asic trident2 --target saivs --sai-server=172.17.0.3 -v -k "test_l2_basic"
```